### ISSUE TITLE
Custom max hit sound plays twice

### ISSUE THREAD

**Issue by VanessaMagick** [Hit-Sounds/hit-sounds#5]
I'm having a persistent issue where a custom max.wav will always play twice when triggered. The exact same file functions totally fine as any other hitsound, and any other file I use produces the same effect.

**Comment by Ferrariic** [comment]
I'll investigate this issue.

**Comment by Ferrariic** [comment]
Could you provide more detail and perhaps a video so that I can see when and where this is occurring? thank you :)

**Comment by VanessaMagick** [comment]
Certainly. In this instance I have two custom hitsounds, normal hits (normal.wav) and max hits (max.wav)

The normal hit sounds usually play correctly, but max hit sounds usually play twice. They also tend to double up the sound of the next hit, if it does hit.

Both files used in this clip are less than 600ms, incidentally.

https://user-images.githubusercontent.com/35085611/225730472-aed50484-3557-46c7-9420-c586940ec8f4.mp4

**Comment by blake2create** [comment]
Having the same issue with the max hit sound playing twice. Has anyone figured out a fix for this?

**Comment by VanessaMagick** [comment]
Issue still persists, I'm afraid.

**Comment by SyntaxBleach** [comment]
Having the same issue here, would assume the fix would be to make the sound effect file last longer by adding extra silent time?

Edit: Tried at around 10 seconds and nope, not working still, hopefully it can be fixed quickly @Ferrariic , this definitely has quite a bit of potential, hope to see this fixed! 🙂
